pages/curriculum-workshop-march-8-2016.jsx

```diff
@@ -0,0 +1,83 @@
+var React = require('react');
+var HeroUnit = require('../components/hero-unit.jsx');
+// use this LinkAnchorSwap component for hyperlinks
+var LinkAnchorSwap = require('../components/link-anchor-swap.jsx');
```

Comment:
sorry forgot to tell you @flukeout , could you use `<LinkAnchorSwap to={url}>` for hyperlinks instead? We want to GA track traffic to outbound links and `<LinkAnchorSwap>` does the magic. See the [source code](https://github.com/mozilla/teach.mozilla.org/blob/develop/components/link-anchor-swap.jsx) in case you are curious.